teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md

```diff
@@ -0,0 +1,393 @@
+   -connectorSpecificSettings (New-Object Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.ConnectorSpecificUkgDimensionsSettingsRequest `
+        -Property @{
+            apiUrl = "https://www.contoso.com/api"
+            ssoUrl = "https://www.contoso.com/sso"
+            appKey = "PlaceholderForAppKey"
+            clientId = "Es5Q2fB4PXweCyto6Ms6J2kRB7uwAc3W"
+            clientSecret = "PlaceholderForClientSecret"
+            LoginUserName = "PlaceholderForUsername"
+            LoginPwd = "PlaceholderForPassword"
+    })
+PS C:\> $result | Format-List
+```
+
+```output
+
+ConnectorId                              : 95BF2848-2DDA-4425-B0EE-D62AEED4C0A0
+ConnectorSpecificSettingAdminApiUrl      :
+ConnectorSpecificSettingApiUrl           : https://www.contoso.com/api
+ConnectorSpecificSettingAppKey           :
+ConnectorSpecificSettingClientId         : Es5Q2fB4PXweCyto6Ms6J2kRB7uwAc3W
+ConnectorSpecificSettingCookieAuthUrl    :
+ConnectorSpecificSettingEssApiUrl        :
+ConnectorSpecificSettingFederatedAuthUrl :
+ConnectorSpecificSettingRetailWebApiUrl  :
+ConnectorSpecificSettingSiteManagerUrl   :
+ConnectorSpecificSettingSsoUrl           : https://www.contoso.com/sso
+CreatedDateTime                          : 06/04/2023 11:05:39
+Etag                                     : "3100fd6e-0000-0400-0000-642ea7840000"
+Id                                       : a2d1b091-5140-4dd2-987a-98a8b5338744
+LastModifiedDateTime                     : 06/04/2023 11:05:39
+Name                                     : Cmdlet test connection - updated
+TenantId                                 : 3FDCAAF2-863A-4520-97BA-DFA211595876
+
+```
+
+Updates the instance with the specified -ConnectionId. Returns the object of updated connection.
+
+In case of error, we can capture the error response as following:
+
+* Hold the cmdlet output in a variable: `$result=<CMDLET>`
+
+* To get the entire error message in Json: `$result.ToJsonString()`
+
+* To get the error object and object details: `$result, $result.Detail`
+
+## PARAMETERS
+
+### -Body
+The request body 
+
+```yaml
+Type: IUpdateWfmConnectionRequest
+Parameter Sets: Set, SetViaIdentity
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Break
+Wait for .NET debugger to attach.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ConnectionId
+The WFM connection id for the instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```

Comment:
```suggestion
The WFM connection ID for the instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```